pkg/tsdb/opentsdb/opentsdb.go

```diff
@@ -218,3 +216,15 @@ func (e *OpenTsdbExecutor) buildMetric(query *tsdb.Query) map[string]interface{}
 
 	return metric
 }
+
+func (e *OpenTsdbExecutor) createSeriesLabel(res OpenTsdbResponse) string {
```

Comment:
dont you need to handle the alias pattern interpolation here? This is done in the frontend code here, https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/opentsdb/datasource.js#L348